include/psa/crypto_extra.h

```diff
@@ -1901,6 +1901,342 @@ static inline struct psa_pake_operation_s psa_pake_operation_init( void )
     return( v );
 }
 
+/**
+ * \brief The context for PSA interruptible signing.
```

Comment:
For Mbed TLS's sake, we need to document on all the new definitions that that this is still a beta API, subject to change. It is not bound by our usual interface stability promises. It will only leave beta after we've implemented it.